### PR TITLE
Tasks for fgbio's GroupReadsByUmi and CallMolecularConsensusReads.

### PR DESCRIPTION
**Pull request by tfenne**

### CODE REVIEW

**Comment by tfenne** [comment]
@nh13 just two new task classes

File: tasks/src/main/scala/dagr/tasks/fgbio/GroupReadsByUmi.scala [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+import scala.collection.mutable.ListBuffer
+
+/** Task to invoke GroupReadsByUmi. */
+class GroupReadsByUmi(val in:  PathToBam,
+                      val out: PathToBam,
+                      val rawTag: Option[String] = None,
+                      val assignTag: String,
+                      val minMapQ: Option[Int] = None,
+                      val strategy: String = "adjacency",
```
Comment (nh13) [inline tasks/src/main/scala/dagr/tasks/fgbio/GroupReadsByUmi.scala]: Perhaps here it makes more sense to have an enum, similar to how we have `DownsamplingStrategy` in `DownsampleSam`.

**Comment by nh13** [comment]
One minor comment, fix up (or not), I would though, and then merge.

**Comment by tfenne** [comment]
I've done as you suggested but don't love it.  Should probably, at some point, make it a Java enum in `fgbio` and depend on that here etc.

**Pull request merged**